library/src/test/java/com/novoda/downloadmanager/FakeDownloadsPersistence.java

```diff
@@ -73,4 +73,9 @@ public void delete(DownloadBatchId downloadBatchId) {
     public void update(DownloadBatchId downloadBatchId, DownloadBatchStatus.Status status) {
 
     }
+
+    @Override
+    public void update(DownloadBatchId downloadBatchId, boolean notificationSeen) {
+
```

Comment:
All these empty method could use a `// no-op` comment, just to specify that we didn't forget about these